### DISCUSSION TITLE
Unable to start bug - person_centric_id

### DISCUSSION THREAD

**Discussion by nathanjmassey** [ReagentX/imessage-exporter#188] [Q&A] [answered]
I'm getting an error message when trying to run imessage-exporter. Any ideas on what the problem could be? Details below,

System: macOS High Sierra 10.13.6
Messages.app: v11.0

I ran `cargo run imessage-exporter` to get the package installed.

I then ran `imessage-exporter -d` and got the below error,

```
Building cache...
[1/4] Caching chats...
[2/4] Caching chatrooms...
[3/4] Caching participants...
[4/4] Caching reactions...
Cache built!

iMessage Database Diagnostics

Processing...Unable to start: Failed to parse row: no such column: person_centric_id in SELECT COUNT(DISTINCT person_centric_id) FROM handle WHERE person_centric_id NOT NULL at offset 22
```

**Comment by ReagentX** [comment, marked as answer]
High Sierra is very old; this software targets the latest releases for MacOS and iOS only.

**Reply by ReagentX** [reply]
Specifically, this older table does not have the `person_centric_id`, which is required for deduplicating contacts.

**Reply by ReagentX** [reply]
This is now resolved by #250!
